### ISSUE TITLE
Define onboarding whitelist merging strategy

### ISSUE THREAD

**Issue by libremente** [italia/developers-italia-backend#86]
The crawler's whitelist is a combination of at least 2 YAML files: the one present here https://github.com/italia/developers-italia-backend/blob/master/crawler/whitelist/publishers.yml.example and the one filled by the onboarding procedure. 
An option might be to parse everything contained in the `whitelist` folder, so a symlink should be enough. However, this approach would require to change the structure of the YAML file produced by the onboarding. 
In fact, backend's YAML:
```
- id: "Comune di Bagnacavallo"
  codice-iPA: "c_a547"
  orgs:
    - "https://github.com/gith002"
  repos:
    - "https://github.com/gith002/foobar"
```
whilst onboarding YAML:
```
registrati: 
    - 
      timestamp: "2019-05-27T09:45:00.770Z"
      ipa: "c_a123"
      url: "https://github.com/undefined"
      pec: "protocollo.comunemaramao@pec.it"
```
Probably the best approach requires to parse the onboarding whitelist, extract the information needed and then populate the crawler's whitelist. What do you think? @sebbalex @alranel

**Comment by libremente** [comment]
I guess this has been fixed by 9bd3668ea98e2c2a16f52681929adf67a8d37b48, isn't it @alranel ?

**Comment by bfabio** [comment]
Fixed by 9bd3668ea98e2c2a16f52681929adf67a8d37b48.
